lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -977,7 +980,15 @@ public int getFrame() {
   }
 
   public void setProgress(@FloatRange(from = 0f, to = 1f) float progress) {
-    userActionsTaken.add(UserActionTaken.SET_PROGRESS);
+    setProgressInternal(true, progress);
+  }
+
+  private void setProgressInternal(
```

Comment:
Could you swap the parameter order here?

Reply:
@gpeal Done

Reply:
I've rebased on master as well